openstf/model/split_energy.py

```diff
@@ -12,6 +12,8 @@
 
 import openstf.monitoring.teams as monitoring
 
+COEF_MAX_PCT_DIFF = 0.3
```

Comment:
The variable says 'pct' (= Percent?), but you actually use a fraction and not a percentage

Reply:
you are correct, i changed the name to `COEF_MAX_FRACTION_DIFF`